lib/bridge/MatrixHandler.js

```diff
@@ -770,6 +769,68 @@ MatrixHandler.prototype._onMessage = Promise.coroutine(function*(req, event) {
     yield Promise.all(promises);
 });
 
+MatrixHandler.prototype._sendIrcAction = Promise.coroutine(
+    function*(req, ircRoom, ircUser, ircAction, event) {
+    let self = this;
+
+    if (event.content.msgtype !== "m.text" ||
+        !(ircUser.unsafeClient && ircUser.unsafeClient.wouldSend)) {
+        yield self.ircBridge.sendIrcAction(ircRoom, ircUser, ircAction);
+    }
+    else {
+        let text = event.content.body;
+        let wouldSend = ircUser.unsafeClient.wouldSend(ircRoom.channel, text);
+        let lineLimit = 2;
+        if (wouldSend.length > lineLimit) {
```

Comment:
`wouldSend` is a misleading variable name - I intuitively thought it would be a `boolean` but it turns out it is an `array`?!